### DISCUSSION TITLE
Purpose of the "ua-parser.pack.js" file?

### DISCUSSION THREAD

**Discussion by knubew** [faisalman/ua-parser-js#675] [General]
Hello,

I think I will switch from "mobile-detect" (https://github.com/hgoebl/mobile-detect.js) to "ua-parser-js" (1.0.36).

- Could someone please briefly explain to me the purpose of the "ua-parser.pack.js" file?
- Do I also need this file if I want to include "ua-parser.min.js" in my project?

Thanks in advance.

**Comment by faisalman** [comment]
Both are a minified version of the main UAParser.js file, with the .pack.js having less bytes than .min.js in exchange of readability. You only need to include one of them.

**Reply by knubew** [reply]
Thank you very much for your quick response and the explanation.
